third_party/bigframes_vendored/pandas/core/frame.py

```diff
@@ -2292,6 +2354,71 @@ def merge(
             rows will be matched against each other. This is different from usual SQL
             join behaviour and can lead to unexpected results.
 
+        **Examples:**
```

Comment:
I feel like it would be best to borrow the examples from https://pandas.pydata.org/docs/reference/api/pandas.DataFrame.merge.html, WDYT?